daemon/src/maker.rs

```diff
@@ -113,7 +113,7 @@ async fn main() -> Result<()> {
 
     let (order_feed_sender, order_feed_receiver) = watch::channel::<Option<Order>>(None);
     let (wallet_feed_sender, wallet_feed_receiver) = watch::channel::<WalletInfo>(wallet_info);
-    let (settlement_feed_sender, settlement_feed_receiver) =
+    let (update_cfd_feed_sender, update_cfd_feed_receiver) =
```

Comment:
These renames better fit into the previous commit if you want to keep them separate.

Reply:
If you are fine with it then I'm happy to just squash both commits.

Reply:
Sure!